lib/command_runner.dart

```diff
@@ -49,17 +49,20 @@ class CommandRunner<T> {
 
   /// Returns [usage] with [description] removed from the beginning.
   String get _usageWithoutDescription {
+    const usagePrefix = "Usage:";
     var usage = '''
-Usage: $invocation
+$usagePrefix ${_wrap(invocation, hangingIndent: usagePrefix.length)}
 
-Global options:
+${_wrap('Global options:')}
 ${argParser.usage}
 
-${_getCommandUsage(_commands)}
+${_getCommandUsage(_commands, lineLength: argParser.usageLineLength)}
 
-Run "$executableName help <command>" for more information about a command.''';
+${_wrap('Run "$executableName help <command>" for more information about a command.')}''';
```

Comment:
There's enough interpolation and stuff going on in this literal that it's probably worth switching to a StringBuffer and building it up imperatively.